docker/release.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 set -ex
 
-vt_base_version='v16.0.0'
-debian_versions='buster  bullseye'
+vt_base_version='v17.0.0'
+debian_versions='buster  bullseye  bookworm'
```

Comment:
I don't have a strong opinion either way. There's minimal issue leaving it building, and you have to explicitly opt into the buster image, since bullseye is the default. 

Even though bookworm is available now, other providers like Percona won't be adding support until the end of the year, and I imagine many will give it some time before upgrading, if that impacts the decision at all.

https://jira.percona.com/browse/PS-8807
https://forums.percona.com/t/debian-bookworm-has-been-released/22734

Reply:
Related question, should we build v17.0.0 with bookworm? If not, v17.0.1 would be the first release to have it, whenever we end up making it

Reply:
@derekperkins after re-reading the `buster` support documentation, LTS supports will stop in June 2024, meaning that the next vitess release (`v18.0`, will be released in October) will have a `buster` build that is potentially deprecated for several months after June 2024 - making me want to remove the `buster` build on `main` for future releases.

Regarding the `vt_base_version`, let's move it to `v18.0.0-SNAPSHOT` on `main` - It is already on `v17.0.1-SNAPSHOT` on the `release-17.0` branch.

I don't think we need to backport this change to `release-17.0` unless someone asks us to build `v17.0` on `bookworm`.

Reply:
Additionally, @dbussink pointed out we should update the `lite` docker images to use `bullseye` too as the default debian image. I can open a follow-up PR for that.

Reply:
Should I change the default for v18 to bookworm?

Reply:
We can try, although @dbussink and I are not sure if Oracle ships older MySQL version for a new debian release

Reply:
None of these containers have a dependency on MySQL, so I think that's only relevant for the lite containers

Reply:
okay awesome, in that case let's change the default to `bookworm` for v18.0

Reply:
changes made and entries added to the v18 changelog